src/Illuminate/Routing/Router.php

```diff
@@ -674,6 +677,21 @@ public static function prepareResponse($request, $response)
     }
 
     /**
+     * Create an illuminate response from a given base response object.
+     *
+     * @param  mixed  $response
+     * @return \Illuminate\Http\Response
+     */
+    private static function createResponseFromBase($response)
+    {
+        return (new Response(
+            $response->getContent(),
```

Comment:
Also why would you access it via $response->getOriginalContent and  $response->baseResponse->getOriginalContent in another case